R/write_file.R

```diff
@@ -0,0 +1,22 @@
+#' Writes text to file
+#'
+#' This function is a wrapper around [`brio::write_lines()`].
+#' It also supports verboe output, similar to [`usethis::write_over()`],
+#' controlled by the `quiet` parameter.
+#' @param text Character vector containing text to write.
+#' @param path A string giving the file path to write to.
+#' @param search_root_from A string giving path to a package subfolder,
+#' which is used to find package root and produce message for the user.
```

Comment:
I don't entirely understand what this parameter does from this documentation. Is this only for message generation? If yes, say so explicitly. "This parameter only affects any messages generated to the user. It has no effect on where the file is written."